livekit_analytics.proto

```diff
@@ -34,6 +34,8 @@ message AnalyticsStat {
   int32 pli_count = 13;
   int32 fir_count = 14;
   string room_name = 15;
+  float rating = 16;
```

Comment:
nit: do we use the terminology rating or score? can we prefix it with `connection_` to disambiguate?

Reply:
i jupdated it to score, will prefix connection_